src/core/annotation/AnnotationManager.ts

```diff
@@ -151,19 +151,37 @@ class AnnotationManager {
    * Parse an array of annotation data
    * @param {object[]} data An array of annotation data
    */
-  public parse(data: Array<Record<string, any>>): Annotation[] {
+  public parse(data: {
+    baseFov: number;
+    baseDistance: number;
+    aspect: number;
+    items: Array<{
+      meshIndex: number;
```

Comment:
Although the type of meshIndex is a required number, it seems okay to specify optional and null types with 171 line

Reply:
Good point 👍